src/PIL/TiffTags.py

```diff
@@ -18,10 +18,18 @@
 ##
 from __future__ import annotations
 
-from collections import namedtuple
+from typing import NamedTuple
 
 
-class TagInfo(namedtuple("_TagInfo", "value name type length enum")):
+class _TagInfo(NamedTuple):
+    value: int
+    name: str
+    type: int
+    length: int
+    enum: dict[int, str]
```

Comment:
```suggestion
    value: int | None
    name: str
    type: int | None
    length: int | None
    enum: dict[int, str] | None
```

It would seem that `value`, `type`, `length` and `enum` can all be `None`
https://github.com/python-pillow/Pillow/blob/58c4c757a2e969f52d23586e731571887cad2db0/src/PIL/TiffTags.py#L27-L28